lib/active_merchant/billing/gateways/decidir.rb

```diff
@@ -99,7 +99,8 @@ def scrub(transcript)
         transcript.
           gsub(%r((apikey: )\w+)i, '\1[FILTERED]').
           gsub(%r((\"card_number\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"security_code\\\":\\\")\d+), '\1[FILTERED]')
+          gsub(%r((\"security_code\\\":\\\")\d+), '\1[FILTERED]').
+          gsub(%r((\"emv_issuer_data\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
Is the `emv_issuer_data` something new that comes through in the response?

Reply:
I'm not sure. I think so -- it seems to just be the security code repeated which causes the existing scrub test to fail. However, I didn't see anything about it in the docs when I looked. So not exactly sure what the reasoning behind it is.